libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -140,13 +167,16 @@ export class DefaultConfigurationSettingsConverter implements ConfigurationSetti
  * @internal
  * */
 class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettingsConverter {
-  featureManagementKeyWord: string;
-  enabledForKeyWord: string;
-  constructor(featureManagementKeyWord: string, enabledForKeyWord: string) {
-    this.featureManagementKeyWord = featureManagementKeyWord;
-    this.enabledForKeyWord = enabledForKeyWord;
+  legacySchemaFeatureManagementKeyWord: string;
+  foundMsFeatureManagement: boolean;
+  ajv: Ajv;
+
+  constructor(legacySchemaFeatureManagementKeyWord: string, foundMsFeatureManagement: boolean) {
+    this.legacySchemaFeatureManagementKeyWord = legacySchemaFeatureManagementKeyWord;
+    this.foundMsFeatureManagement = foundMsFeatureManagement;
+    this.ajv = new Ajv();
 
-    if (!this.featureManagementKeyWord || !this.enabledForKeyWord) {
+    if (!this.legacySchemaFeatureManagementKeyWord && !this.foundMsFeatureManagement) {
```

Comment:
We can also pass `foundLegacySchema` then on line 179 can be simplified to `if (!this.foundLegacySchema && !this.foundMsFeatureManagement)`

Reply:
This would be easier to read. But I was thinking we already pass in the legacySchemaKeyWord to the function( we use it get the other keywords and get the flags from the legacy schema) then there is no need to pass `foundLegacySchema ` because if the legacySchemaKeyWord is null the foundLegacySchema will always be false